gulpfile.js

```diff
@@ -61,6 +61,13 @@ gulp.task('build:assets',
           false,
           t.getTask('assets'));
 
+
+// Moves all the assets to the build
+gulp.task('build:accessibility',
```

Comment:
yap, that's better. Since I'm there, should I create a ```'test' --> ['test:accessibility'] ``` that could be extended in the future with (for example) 'test:performence', 'test:unit', 'test:e2e', 'test:browser-compatability'?